src/components/nav-bar/nav-bar.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 1em;
+  transition: all 1s ease-in-out;
+}
+
+body.dark oui-nav-bar {
+  display: block;
+  list-style-type: none;
+  padding: 3px 5px;
+  margin-top: 3em;
+  left: 50%;
+  right: 50%;
+  transform: translate(-50%, -50%);
+  overflow: hidden;
+  background-color: rgb(40, 51, 68);
+  position: relative;
+  top: 0;
+  width: inherit;
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
+  border-radius: 50px;
+  transition: all 1s ease-in-out;
+}
+
+oui-nav-bar {
+  display: block;
+  list-style-type: none;
+  padding: 3px 5px;
+  margin-top: 3em;
+  left: 50%;
+  right: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
This positioning seems very specific. Make sure this nav bar can be use outside of your specific test scenario.

Reply:
Will do.